src/components/SortorderDropdown.vue

```diff
@@ -20,86 +20,75 @@ License along with this library.  If not, see <http://www.gnu.org/licenses/>.
 -->
 
 <template>
-	<div>
-		<div v-click-outside="closeMenu" class="app-navigation-entry-utils reactive" @click="toggleMenu">
-			<div :title="$t('tasks', 'Change sort order')" class="app-navigation-entry-utils-menu-button">
-				<button class="sortorder-dropdown-button">
-					<span :class="sortOrderIcon" class="icon icon-bw" />
-					<span :class="sortDirection ? 'icon-sort-up' : 'icon-sort-down'" class="icon icon-bw sort-indicator" />
-				</button>
-			</div>
-		</div>
-		<div :class="{'open': menuOpen}" class="app-navigation-entry-menu bubble sortorder-dropdown reactive">
-			<ul>
-				<li v-for="order in orders"
-					:key="order.id"
-					v-tooltip="{
-						placement: 'left',
-						content: order.hint,
-						delay: { show: 500, hide: 0 }
-					}"
-					:class="{active: sortOrder == order.id}"
-					@click="setSortOrder(order.id)"
-				>
-					<a>
-						<span :class="order.icon" class="icon icon-bw" />
-						<span class="label">
-							{{ order.text }}
-						</span>
-						<span :class="sortDirection ? 'icon-sort-up' : 'icon-sort-down'" class="icon icon-bw sort-indicator" />
-					</a>
-				</li>
-			</ul>
-		</div>
-	</div>
+	<Actions class="sortorder reactive"
+		:title="$t('tasks', 'Change sort order')"
+		:default-icon="sortOrderIcon"
+		menu-align="right"
+	>
```

Comment:
Are you using the nextcloud eslint config? This should not be valid :)

Reply:
Not yet 🙈 But I guess this is something for another PR. Issue is #837

Reply:
What would be wrong here?

Reply:
`>` should not be alone on a new line :thinking:

Reply:
https://github.com/nextcloud/eslint-config-nextcloud/pull/1
https://github.com/skjnldsv/vueexample/blob/8a8324b9c62a0c719b3f09b7d3be0ce85ee0507a/.eslintrc.js

Reply:
Ah, I remember we had a discussion about this in https://github.com/nextcloud/eslint-config-nextcloud/pull/8#issuecomment-546364530

For me this looks way cleaner on a new line, since I quickly see where the label element ends
``` html
<label
    class="button"
>
    {{ Text }}
</label>
```
where as with
``` html
<label
    class="button">
    {{ Text }}
</label>
```
it looks as if `{{ Text }}` belongs to the `label` element.

Am I allowed to keep it like this 😃

Reply:
Btw. this is not introduced by this PR, but like this in Tasks all over the place.

Reply:
> Am I allowed to keep it like this?

I mean, I won't be the eslint police here :grin: 
But we talked and found a consensus, so I think it would be nice to have our nc apps unified in the code styling. https://github.com/nextcloud/eslint-config-nextcloud/pull/8#issuecomment-546633528

Reply:
Ok, let's discuss this in #837, this is nothing for this PR anyway.